src/internal/kopia/upload.go

```
@@ -141,6 +142,18 @@ type corsoProgress struct {
 	mu         sync.RWMutex
 	totalBytes int64
 	errs       *fault.Bus
+	// expectedIgnoredErrors is a count of error cases caught in the Error wrapper
+	// which are well known and actually ignorable.  At the end of a run, if the
+	// manifest ignored error count is equal to this count, then everything is good.
+	expectedIgnoredErrors int
```

@ashmrtn How's this approach for tallying ignored errors?